tests/countries/test_taiwan.py

```diff
@@ -201,12 +263,33 @@ def test_lunar_new_year(self):
         self.assertHolidayName(f"{name}（慶祝）", obs_dt)
         self.assertNoNonObservedHoliday(obs_eve_dt, obs_dt)
 
+    def test_taoism_day(self):
+        name = "道教節"
+
+        # Workdays.
+        self.assertHolidayName(
+            name,
+            self.wrk_holidays,
+            "2015-02-19",
+            "2016-02-08",
+            "2017-01-28",
+            "2018-02-16",
+            "2019-02-05",
+            "2020-01-25",
+            "2021-02-12",
+            "2022-02-01",
+            "2023-01-22",
+            "2024-02-10",
+        )
+        self.assertHolidayName(name, self.wrk_holidays, range(2001, 2050))
+        self.assertNoHolidayName(name, self.wrk_holidays, range(1998, 2001))
+        self.assertNoHolidayName(name, range(1998, 2050))
```

Comment:
```suggestion
        self.assertNoHolidayName(name)
```
It's a bit simpler. :)

Reply:
Fixed.